src/main/java/net/sf/jabref/logic/integrity/IntegrityCheck.java

```diff
@@ -10,7 +11,11 @@
 import net.sf.jabref.model.entry.ParsedFileField;
 
 import java.io.File;
-import java.util.*;
```

Comment:
I use Eclipse, so it should behave as we want it to if nothing else... I changed it manually though.